readme.md

```diff
@@ -509,7 +558,49 @@ Any `Defaults` key changes made within the closure will not propagate to `Defaul
 
 Get/set a `Defaults` item and also have the view be updated when the value changes.
 
-This is only implemented for `Defaults.Key`. PR welcome for `Defaults.NSSecureCoding` if you need it.
+### `Defaults.migration(keys..., to: Version)`
+
+```swift
+Defaults.migration<T: Defaults.Serializable & Codable>(keys..., to: Version)
+Defaults.migration<T: Defaults.NativeType>(keys..., to: Version)
+```
+
+Type: `func`
+
+Migrate the given keys to the specific version.
+
+You can specify up to 10 keys. If you need to specify more, call this method multiple times.
+
+#### `Defaults.NativeType`
+
+```swift
+protocol DefaultsNativeType: Defaults.Serializable {
+	associatedtype CodableForm: Defaults.CodableType
+}
+```
+
+Type: `protocol`
+
+Represents the type after migration.
+
+It should have a associated type name `CodableForm` which protocol conform to `Codable`.
+
+#### `Defaults.CodableType`
+
+```swift
+protocol DefaultsCodableType: Codable {
+	associatedtype NativeForm: Defaults.NativeType
+	func toNative() -> NativeForm
+}
+```
+
+Type: `protocol`
+
+Represents the type before migration.
+
+It should have an associated type name `NativeForm` which is the type we want it to store in `UserDefaults`.
+
+And it also have a `toNative()` function to convert itself into `NativeForm`.
```

Comment:
I don't think this type should be in the readme. It's only meant for migration. Users shouldn't have to care about it. It will eventually be removed. I think the source code doc comment should also mention this fact.